scripts/clean-node-modules.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+# we have something weird going on, where installing via `file:../..` doesn't take "files" in account, and thus copies recursively. This causes cache to take many minutes and us to time-out. The long-term solution is to use prevent the recursive copying too, which will make the install faster.
+
+mkdir /tmp/empty
```

Comment:
I'm not sure if this works in other environment like Windows, but let's fix it in another PR in another time.

Reply:
netlify is only macOS so that's no big deal IMO

Reply:
I meant @yannickcr ... 😆

Reply:
I'm on Linux, I'm not that barbaric :rofl:

Reply:
ohhh my bad 🤪